### DISCUSSION TITLE
Docs - move, copy or reference "Livewire's "hybrid" JavaScript functions" section to/in https://livewire.laravel.com/docs/javascript page

### DISCUSSION THREAD

**Discussion by jeffchown** [livewire/livewire#9005] [3. Feature ideas]
Hi @calebporzio - I had to look up the hybrid Javascript functions/Evaluating one-off JavaScript expressions today and it took some time to find as I had forgotten it 'lives' on the Actions page (https://livewire.laravel.com/docs/actions).

Might be me ;) but shouldn't it live, or at least be linked to, on the Javascript page (https://livewire.laravel.com/docs/javascript) to be easier to find?

Thanks!

**Comment by joshhanley** [comment]
@jeffchown PRs to update the docs are welcome! 🙂

**Reply by jeffchown** [reply]
Thanks, @joshhanley - I'll see what I can come up with.

**Reply by lrljoe** [reply]
If you're updating those docs, there's a couple of examples that show what not to do, with a comment at the top of the snippet.
I can't help but feel like adding an extra comment immediately above the dodgy methods might be kind!

To be honest, I think adding a different "bad code snippet" background colour (e.g. a red) might make it really clear when the docs are referencing a "don't do this" or "this is the old way", but that's one for @calebporzio and @joshhanley to decide on.

**Reply by joshhanley** [reply]
@lrljoe that doesn't seem related to what @jeffchown has raised here, so that would need to be a separate PR 🙂
